redis.conf

```diff
@@ -1697,6 +1697,30 @@ slowlog-max-len 128
 # "CONFIG SET latency-monitor-threshold <milliseconds>" if needed.
 latency-monitor-threshold 0
 
+################################ LATENCY STATS ##############################
+
+# The Redis extended latency stats track per command latencies and enable
+# exporting the percentile distribution and cumulative latency distributions,
+# via the INFO LATENCYSTATS command.
+# We have two groups of outputs on INFO LATENCYSTATS:
+#   - latency by percentile distribution.
+#   - cumulative distribution of latencies.
+# Using the cumulative distribution of latencies you can merge several stats
+# from different nodes to calculate aggregate metrics. Note that percentile
+# distribution is not mergeable between nodes.
+#
+# By default, the extended latency monitoring is enabled since the overhead
+# of keeping track of the command latency is very small.
+# if you don't want to track extended latency metrics, you can easily disable it
+# at runtime using the command:
+# "CONFIG SET latency-track no" if needed.
+# latency-track yes
+#
+# By default the exported latency percentiles are the p50, p99, and p999.
+# You can alter them at runtime using the command:
+# CONFIG SET latency-track-percentiles "0.0 50.0 100.0"
```

Comment:
I feel like this is self explanatory information about configs, not sure it's needed.